tests/S3/Crypto/S3EncryptionMultipartUploaderTest.php

```diff
@@ -10,7 +10,7 @@
 use Aws\Test\Crypto\UsesMetadataEnvelopeTrait;
 use GuzzleHttp\Psr7;
 
-class S3EncryptionMultipartUploaderTest extends \PHPUnit_Framework_TestCase
```

Comment:
Instead of specifying the whole namespace in the `extends`, can you port this to a `use` statement and just reference the class name?

```php
use PHPUnit\Framework\TestCase;

class S3EncryptionMultipartUploaderTest extends TestCase
```

Reply:
Sure, I did it d14c8e3